package/score-worker/src/main/java/org/openscore/schema/WorkerBeanDefinitionParser.java

```diff
@@ -70,7 +70,9 @@
 			new ConfValue().NAME("outBufferInterval").DEFAULT(100L),
 			new ConfValue().NAME("keepAliveInterval").DEFAULT(10000L),
 			new ConfValue().NAME("configRefreshInterval").DEFAULT(1000L),
-			new ConfValue().NAME("statisticsInterval").DEFAULT(1000L)
+            new ConfValue().NAME("statisticsInterval").DEFAULT(1000L),
```

Comment:
regarding the schedulers I prefer that the consumer,that uses the data, would run the jobs and not score.